integrations/symfony-bundle.rst

```diff
@@ -342,6 +343,30 @@ The only steps they need is ``require`` one of the adapter implementations in
 their projects ``composer.json`` and instantiating the ``HttplugBundle`` in
 their kernel.
 
+Mock responses in functional tests
```

Comment:
we use the convention of all caps in titles, this should be "Mock Responses In Functional Tests"